### PR TITLE
chore: code style

### PR DESCRIPTION
**Pull request by xiaohuoni**
##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/umi/ISSUE_URL

### CODE REVIEW

File: .github/commit-convention.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ The **header** is mandatory and the **scope** of the header is optional.
 
 ### Revert
 
-If the commit reverts a previous commit, it should begin with `revert: `, followed by the header of the reverted commit. In the body, it should say: `This reverts commit <hash>.`, where the hash is the SHA of the commit being reverted.
+If the commit reverts a previous commit, it should begin with `revert:`, followed by the header of the reverted commit. In the body, it should say: `This reverts commit <hash>.`, where the hash is the SHA of the commit being reverted.
```
Comment (sorrycc) [inline .github/commit-convention.md]: `revert: ` 这个没问题。

File: .github/commit-convention.md [hunk 2]
```diff
@@ -74,18 +74,16 @@ The scope could be anything specifying the place of the commit change. For examp
...
-* don't capitalize the first letter
-* no dot (.) at the end
+- use the imperative, present tense: "change" not "changed" nor "changes"
+- don't capitalize the first letter
+- no dot (.) at the end
 
 ### Body
 
-Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes".
-The body should include the motivation for the change and contrast this with previous behavior.
```
Comment (sorrycc) [inline .github/commit-convention.md]: 这个不要合成一行。

File: .github/commit-convention.md (hunk 1 above)
Comment (xiaohuoni) [reply .github/commit-convention.md]: MD038/no-space-in-code: Spaces inside code span elementsmarkdownlint(MD038)

Comment (xiaohuoni) [reply .github/commit-convention.md]: MD038/no-space-in-code: Spaces inside code span elementsmarkdownlint(MD038)
https://github.com/DavidAnson/markdownlint/blob/v0.18.0/doc/Rules.md#md038

Comment (xiaohuoni) [reply .github/commit-convention.md]: 不写空格的话，在 `verify commit` 的时候会被拒绝。我觉得文档这里还是按照规范去掉空格会好一点。

**Pull request merged**